website/components/Layout/index.js

```diff
@@ -0,0 +1,55 @@
+import Head from 'components/Head';
+import Navbar from 'components/Navbar';
+import StyledLink from 'components/StyledLink';
+import PropTypes from 'prop-types';
+import { Layout, ContentLayout } from '@strapi/design-system/Layout';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { useTheme } from '@strapi/design-system/ThemeProvider';
+
+const AppLayout = ({ children, meta, navigationContent }) => {
+  const theme = useTheme();
+  return (
+    <Box background="neutral100">
+      <Layout sideNav={<Navbar navigationContent={navigationContent} />}>
+        <Head meta={meta} />
+        <Box padding={4}>
+          <ContentLayout>
+            <Stack as="main" size={4}>
+              {children}
+            </Stack>
```

Comment:
Can you use the Main component of the design system please.

`import { Main } from '@strapi/design-system/Main';`

```suggestion
         <Main>
            <Stack size={4}>
              {children}
            </Stack>
         </Main>
```